templates/index.twig

```diff
@@ -38,7 +38,6 @@
                  title="{{ 'VisitorGenerator_ChoiceYes'|translate|e('html_attr') }}">
             </div>
 
-            <input type="hidden" value="{{ token_auth }}" name="token_auth"/>
```

Comment:
wasn't needed and caused Matomo for WordPress to redirect to login page because there was something wrong with it